SteakBot.Core/ThreadLocalRandom.cs

```diff
@@ -0,0 +1,44 @@
+﻿using System;
+using System.Threading;
+
+namespace SteakBot.Core
+{
+    // https://codeblog.jonskeet.uk/2009/11/04/revisiting-randomness/
+    /// <summary>
+    /// Convenience class for dealing with randomness.
+    /// </summary>
+    public static class ThreadLocalRandom
+    {
+        /// <summary>
+        /// Random number generator used to generate seeds,
+        /// which are then used to create new random number
+        /// generators on a per-thread basis.
+        /// </summary>
+        private static readonly Random globalRandom = new Random(Guid.NewGuid().GetHashCode());
+        private static readonly object globalLock = new object();
```

Comment:
Static readonly fields should use PascalCase naming.

Reply:
Corrected.